src/staticanalysis/bot/static_analysis_bot/revisions.py

```diff
@@ -164,6 +164,13 @@ def add_improvement_patch(self, analyzer_name, content):
             ImprovementPatch(analyzer_name, repr(self), content)
         )
 
+    def empty(self):
+        '''
+        Empty temporary data in BEFORE mode
+        * improvement patches removed
```

Comment:
Nit: This comment is a little confusing. Maybe simplify it? E.g. `Reset temporary improvement patches (used during BEFORE_AFTER analysis)`.